src/App.tsx

```diff
@@ -1,20 +1,15 @@
-import { PeoplePage } from './components/PeoplePage';
-import { Navbar } from './components/Navbar';
-
+import { Outlet } from 'react-router-dom';
 import './App.scss';
+import NavBar from './components/NavBar';
```

Comment:
That file is called `Navbar.tsx`
```suggestion
import NavBar from './components/Navbar';
```

Reply:
Same for other imports